packages/forma-36-react-components/src/components/Button/Button.css

```diff
@@ -19,7 +19,7 @@
   font-family: var(--font-stack-primary);
   font-size: calc(1rem * (14 / var(--font-base-default)));
   overflow: hidden;
-  transition: all var(--transition-duration-default)
+  transition: background-color var(--transition-duration-default)
```

Comment:
@wichniowski Will this have any impact on the loading spinner or any other prop? If not, feel free to merge!

Reply:
ohhh thanks for the heads up!